### PR TITLE
Profile strings

### PR DESCRIPTION
**Pull request by brianteeman**
This changes the label of the password field in the front end edit profile and the admin your profile screens to clarify that it is only if you want to change the password

image

image

At the same time some strings have been changed from "their" to "your" etc

pr for #21923

### CODE REVIEW

**Comment by ghost** [comment]
@ghost can you please Test this PR for your reported Issue?

**Comment by infograf768** [comment]
~I have tested this item :red_circle: unsuccessfully on 0ceb4a8b512b11241985aaaae1a4c292ec1bad8e~<br /><br />~I totally disagree about changing `this user` to `you` as the Basic Settings of a user in backend are not specifically modified for `you`.
It would only make sense if a user with access to manage users is editing his/her own profile.
For frontend it makes sense.~
<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22045">issues.joomla.org/tracker/joomla-cms/22045</a>.</sub>

**Comment by brianteeman** [comment]
Did you actually test this or read the instructions or look at the screenshots? 
If you had then you would realise that I am not an idiot and that the changes are ONLY on pages where you are dealing with your OWN account.

**Comment by infograf768** [comment]
I did not test it, just looked at the .diff
Will test now.

**Comment by brianteeman** [comment]
Well i guess you didn't read the diff carefully or you would have realised that the changes are in
administrator/.../en-GB.com_admin.ini
and not in
administrator/../en-GB.com_users.ini

I will certainly need to be careful accepting your test reports in the future

**Comment by infograf768** [comment]
> I will certainly need to be careful accepting your test reports in the future

That was a very useful comment. One should always verify test reports from anyone.

**Comment by infograf768** [comment]
modified test

**Comment by brianteeman** [comment]
Again I can only assume that you have not tested it or read the instructions or looked at the screenshots

On the admin page when editing a user on the site - NOTHING has changed
On the admin page when editing YOUR profile - the strings have been changed

**Comment by infograf768** [comment]
Can't you read?
I said above I modified my test.
https://issues.joomla.org/tracker/joomla-cms/22045

**Comment by brianteeman** [comment]
Yes - you said you disagree
> I totally disagree about changing this user to you as the Basic Settings of a user in backend are not specifically modified for you.

But if you really tested you would see you are wrong

chrome_2018-09-08_09-37-25
chrome_2018-09-08_09-38-09
chrome_2018-09-08_09-42-46

chrome_2018-09-08_09-38-27
chrome_2018-09-08_09-38-39
chrome_2018-09-08_09-43-02

**Comment by brianteeman** [comment]
So when you said this you are wrong and cannot have tested or you would not disagree
image

**Comment by infograf768** [comment]
I said I modified my test, stop this game.
screen shot 2018-09-08 at 10 46 01

screen shot 2018-09-08 at 10 47 11

I was not going to delete all my comment above just to please your eyes.
Will bar it now.

**Comment by brianteeman** [comment]
**_It is not for my eyes it is for the benefit of others who would still see your objectiion_**

All you had/have to do is to write a new post saying "I was wrong and I have objection"

**Comment by brianteeman** [comment]
Thank you for correcting your post

File: components/com_users/models/forms/profile.xml [hunk 1]
```diff
@@ -29,12 +29,12 @@
 			size="30"
 			validate="username"
 		/>
-
+
```
Comment (Quy) [inline components/com_users/models/forms/profile.xml]: Remove space

**Pull request closed without merging**